packages/style-loader/index.js

```diff
@@ -1,7 +1,7 @@
 const merge = require('deepmerge');
 
 module.exports = (neutrino, opts = {}) => {
-  const modules = opts.modules || true;
+  const modules = opts.modules === false ? false : opts.modules;
```

Comment:
I think this stops `modules` from defaulting to `true`?

Reply:
Gah right, I'll fix.